app/src/main/java/nu/yona/app/utils/AppUtils.java

```diff
@@ -598,19 +597,12 @@ public static boolean isVPNConnected(Context context)
 
 		String profileUUID = YonaApplication.getEventChangeManager().getSharedPreference().getUserPreferences().getString(PreferenceConstant.PROFILE_UUID, "");
 		VpnProfile profile = ProfileManager.get(context, profileUUID);
-		if (ProfileManager.getLastConnectedVpn() != null && profile != null)
-		{
-			Logger.loge("VPNCheck", "last connected vpn profile UUID: " + ProfileManager.getLastConnectedVpn().getUUIDString());
-			Logger.loge("VPNCheck", "current profile id: " + profile.getUUIDString());
-		}
 		if (VpnStatus.isVPNActive() && ProfileManager.getLastConnectedVpn() == profile)
```

Comment:
Change the if/else into just this:
```
return (VpnStatus.isVPNActive() && ProfileManager.getLastConnectedVpn() == profile);
```

Reply:
I forgot to revert the changes I made earlier. Updated the code. Thanks.